spec/acceptance/convert_spec.rb

```diff
@@ -156,17 +156,28 @@
   context 'when converting to the default template' do
     include_context 'in a new module', 'non_default_template'
 
-    answer_file = File.join('..', 'non_default_template_answers.json')
+    before(:each) do
+      @old_answer_file = ENV['PDK_ANSWER_FILE']
+      ENV['PDK_ANSWER_FILE'] = File.expand_path(File.join('..', 'non_default_template_answers.json'))
+    end
+
+    after(:each) do
+      ENV['PDK_ANSWER_FILE'] = @old_answer_file # rubocop:disable RSpec/InstanceVariable This is fine.
+    end
```

Comment:
We can get rid of the instance variable and make rubocop happy using an `around` hook here instead.

```ruby
around(:each) do |example|
  old_answer_file = ENV['PDK_ANSWER_FILE']
  ENV['PDK_ANSWER_FILE'] = File.expand_path(File.join('..', 'non_default_template_answers.json'))
  example.run
  ENV['PDK_ANSWER_FILE'] = old_answer_file
end
```

Reply:
Fixed